src/js/components/FilterMenu/FilterMenu.js

```diff
@@ -1,16 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { CButton } from '@coreui/react';
 import CIcon from '@coreui/icons-react';
 import { cilSettings } from '@coreui/icons';
+import { FormattedMessage } from 'react-intl';
+import FilterTray from '../FilterTray/FilterTray';
+import LifeStyleTray from '../LifeStyleTray/LifeStyleTray';
 
-const FilterMenu = () => (
-  <div className="filter-resorts mb-4 d-flex flex-row">
-    <CButton color="primary" className="me-2 w-100">Lifestyles</CButton>
-    <CButton color="light" className="ms-2 w-100">
-      <CIcon icon={cilSettings} />
-      &nbsp; Filters
-    </CButton>
-  </div>
-);
+const FilterMenu = () => {
+  const [showFilterTray, setShowFilterTray] = useState(false);
+  const [showLifeStyleTray, setShowLifeStyleTray] = useState(false);
+
+  const handleClose = () => {
+    setShowFilterTray(false);
+    setShowLifeStyleTray(false);
+  };
+
+  return (
+    <div className="filter-resorts mb-4 d-flex flex-row">
+      <FilterTray
+        showFilterTray={showFilterTray}
+        handleClose={handleClose}
+      />
+      <LifeStyleTray
+        showLifeStyleTray={showLifeStyleTray}
+        handleClose={handleClose}
+      />
+      <CButton
+        color="primary"
+        aria-roledescription="show-lifestyle-tray"
+        className="me-2 w-100"
+        onClick={() => setShowLifeStyleTray(!showLifeStyleTray)}
```

Comment:
```suggestion
        onClick={() => setShowLifeStyleTray(true)}
```

Is there ever a scenario where this needs to be a two-way toggle?